cluster-api/src/main/java/io/aiven/klaw/clusterapi/services/AivenApiService.java

```diff
@@ -0,0 +1,172 @@
+package io.aiven.klaw.clusterapi.services;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.aiven.klaw.clusterapi.models.AivenAclResponse;
+import io.aiven.klaw.clusterapi.models.AivenAclStruct;
+import io.aiven.klaw.clusterapi.models.ApiResultStatus;
+import io.aiven.klaw.clusterapi.models.ClusterAclRequest;
+import java.util.*;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.*;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
+
+@Service
+@Slf4j
+public class AivenApiService {
+
+  @Value("${klaw.clusters.accesstoken:accesstoken}")
+  private String clusterAccessToken;
+
+  @Value("${klaw.clusters.listacls.api:api}")
+  private String listAclsApiEndpoint;
+
+  @Value("${klaw.clusters.addacls.api:api}")
+  private String addAclsApiEndpoint;
+
+  @Value("${klaw.clusters.deleteacls.api:api}")
+  private String deleteAclsApiEndpoint;
+
+  public Map<String, String> createAcls(ClusterAclRequest clusterAclRequest) {
+    Map<String, String> resultMap = new HashMap<>();
+    RestTemplate restTemplate = getRestTemplate();
+    String projectName = clusterAclRequest.getProjectName();
+    String serviceName = clusterAclRequest.getServiceName();
+
+    LinkedHashMap<String, String> permissionsMap = new LinkedHashMap<>();
+    permissionsMap.put("topic", clusterAclRequest.getTopicName());
+    permissionsMap.put("permission", clusterAclRequest.getPermission());
+    permissionsMap.put("username", clusterAclRequest.getUsername());
+
+    String uri =
+        addAclsApiEndpoint.replace("projectName", projectName).replace("serviceName", serviceName);
+
+    HttpHeaders headers = getHttpHeaders();
+    HttpEntity<LinkedHashMap<String, String>> request = new HttpEntity<>(permissionsMap, headers);
+
+    try {
+      ResponseEntity<String> response = restTemplate.postForEntity(uri, request, String.class);
+      AivenAclResponse aivenAclResponse =
+          new ObjectMapper().readValue(response.getBody(), AivenAclResponse.class);
```

Comment:
`new ObjectMapper()` is a bit heavy to initialize each time.
Since it's thread safe it could be extracted into a static constant